source/use-cases/sensitive-data-encryption.txt

```diff
@@ -13,26 +13,51 @@ Sensitive Data Encryption
 Introduction
 ------------
 
-Cras sed rutrum nisi, vitae tristique risus. Aliquam erat volutpat.
-Donec auctor nisl eget ornare accumsan. Pellentesque lectus tortor,
-consectetur molestie pellentesque ac, blandit non libero.
+If you store information in MongoDB such as confidential personal details, payment information, or proprietary data, you are probably interested in protecting your data from unauthorized access. In addition, the data you store may be subject to governance, privacy, and security compliance mandates. 
+
+MongoDB offers several methods that reduce your risk of a data breach resulting from unauthorized access whether accidental or by a malicious act including:
+
+* `Role-based access control <https://docs.mongodb.com/manual/core/authorization/>`_
+* `Monitoring and logging <https://docs.mongodb.com/manual/core/auditing/>`_
+* `TLS/SSL network transport encryption <https://docs.mongodb.com/manual/core/security-transport-encryption/>`_
+* `Data encryption at rest <https://docs.mongodb.com/manual/core/security-encryption-at-rest/>`_
+* `Client-side field level encryption <https://docs.mongodb.com/manual/core/security-client-side-encryption/>`_
+
+This document guides you through setup and implementation of Client-Side Field Level Encryption (CSFLE), a feature that encrypts document fields specified by a developer, in a example scenario to demonstrate how it can be used to secure your data.
+
+.. note::
+
+  Client-side Field Level Encryption is available starting in MongoDB 4.2 Enterprise only
 
 Problem
 ~~~~~~~
 
-Pellentesque non mi et est congue mattis pretium sit amet mi. Praesent
-nec tortor nec arcu molestie dapibus. Phasellus at nunc ut quam molestie
-finibus. Suspendisse finibus lectus eget nulla eleifend, vel consequat
-leo rhoncus. Phasellus lacinia tortor id aliquet condimentum. Morbi
-sodales est a leo rhoncus, non pretium neque dignissim.
+In this scenario, we secure sensitive data on a Medical Care Management System which stores patients' personal information, appointments, and vital sign measurement reports for a fictitious company, *MedcoMD*. While none of this data should ever be exposed to the public, if a malicious user gained access to data such as the patient's social security number (SSN) or their vital sign measurements, it would be disastrous and damaging to the company.
+
+MedcoMD provides access to this system for two types of users:
+
+1.  Receptionists, who use the system to schedule appointments for Patients, identified by a combination of contact information and their SSN.
+2.  Doctors, who use the system to access and enter Patients' vital sign measurements
```

Comment:
to access the Patient's medical records and enter new vital sign measurements

Reply:
Are we essentially saying that "receptionists shouldn't see sensitive data but doctors need to be able to read and write that data"? I think being explicit about the data access requirements will help clarify the use case, i.e. different clients have different access privileges.

We should also be clear (somewhere) that this won't be accomplished by role-based access permissions (i.e. doctor & receptionist user roles) but instead with permissioned clients (i.e. a reception client and doctor client)

Reply:
Redoing this section to state the following in order to match the final sentence "What can MedcoMD do...":
-What are the use cases the system must satisfy?
-What are the security concerns?

Then, will try to clarify them more in the solution.

Reply:
I wonder if placing equal weight on the two problems this solves -- how to enforce RBAC with FLE and minimizing risk of disclosing sensitive data -- emphasizes the RBAC parts too much as it was not listed in any goals in the [Goals/Non-Goals Scope doc](https://docs.google.com/document/d/1Tl2ftwyUu9g6piJF0yIRdG2eeSc6YL4DV2LqZp4MgxQ/edit#heading=h.ha97b73mwsrj).